ports/superamazingwagonadventure/gameinfo.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gameList>
+  <game>
+    <path>./Super Amazing Wagon Adventure.sh</path>
+    <name>Super Amazing Wagon Adventure</name>
+    <desc>Super Amazing Wagon Adventure is a fast-paced 2d shooting game with a random story. You control a group of three migrants traveling west along a wagon trail. The game consists of short scenes that are sequenced together in a semi-random way to tell a crazy adventure story.</desc>
+    <releasedate>20131017T000000</releasedate>
+    <developer>SparseVector</developer>
+    <genre>Adventure</genre>
+    <image>./superamazingwagonadventure/cover.png</image>
```

Comment:
What's with the long path. Should start with s sure but can be shortened no?
superwagon or whatever.

werenotdoingthesekindoflongnamesbecauseno